### DISCUSSION TITLE
ERM-toimintoon liittyvät käännökset

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha-translations#18] [Yleinen]
En ole "uskaltanut" kääntää Kohan uuteen ERM-toimintoon liittyviä termejä, koska en ole ollut varma, onko niihin jo olemassa vakiintunut termistö. Onko kenelläkään tietoa, onko termistöä olemassa? Jos on, niin pystyisikö osaava tekemään mahdolliset käännökset? :)

**Comment by AnneliO** [comment]
@hanhisalok @inkerih @paivito Mitä mieltä olette tästä ehdotuksesta? :)

**Reply by hanhisalok** [reply]
Meillä tavallaan päätettiin (olisiko ollut se 22.11 päivitys), että sitä ERM-moduulia ei toistaiseksi käännettäisi, mutta todellisuus on se, että senkin toiminnon ominaisuuksia tuli käännettyä runsaasti (osittain pakosti). Taisin itse aika paljon siellä tehdä käännöksiä ja aikalailla omasta päästä (toki yritin noudatella samoja linjoja kuin muissakin Kohan toiminnoissa). Mutta siis ei ole mitään sovittua linjaa ollut meidän puolelta (käsittääkseni).

**Reply by paivito** [reply]
Näihin käännöksiin liittyen. Itse kun en ole tottunut Githubin käyttäjä, niin kysynkin, että missä täällä se listaus jo sovituista käännöksistä on (ennen Koha-Suomen dokumentaatio-sivuilla)?

**Reply by hanhisalok** [reply]
Mutta voidaan kyllä lisätä termistöön sanoja, jos siellä on jotain erityistä, josta pitäisi sopia. Osa termeistä on minusta aika selkeitä käännettäviä. 

Huomiona, että ERM:iä ei meidän puolella mikään kirjastoista vielä oikeasti käytä. Jyväskylässä sitä testailivat, mutta ilmeisesti se ei meille tuollaisenaan tuo nykyiseen elektronisten hallintaan verrattuna juurikaan lisäarvoa.

**Reply by hanhisalok** [reply]
@paivito Siinä yläreunassa pitäisi näkyä kohta wiki, sen alta löytyy termistö.

**Reply by AnneliO** [reply]
Eli käytännössä ei haittaa, vaikka ne loput käännettävät (olikohan niitä 15 kpl) kääntäisi fiiliksellä? Ainahan niitä voi sitten myöhemmin muuttaa, jos ne tuntuu väärältä.

**Reply by hanhisalok** [reply]
Oliko sulla @AnneliO jotain tiettyjä termejä, mitä mietit vai ylipäänsä koko osiotako tarkoitit? 
edit. Voit hyvin kääntää ne nyt, miten parhaaksi näet. Korjataan sitten, jos joku alkaa sitä ominaisuutta oikeasti käyttää ja huomataan tarvetta muokata.

**Reply by AnneliO** [reply]
Ei mitään tiettyjä termejä vaan mietin, että onko noista jo olemassa jonkinlainen tietty sanasto, jota on totuttu käyttämään. Esim. Aggregated full -käännös voisi sanakirjojen mukaan olla jotain tyyliin "yhteen koottu", mutta en tiedä, onko tuosta perinteisesti käytetty jotain muuta termiä kuten "yhteen koonti" tms. Eli multa puuttuu viitekehys, mitä tuo tarkoittaa. :D

**Reply by hanhisalok** [reply]
Eipä ole itsellenikään ERM-maailma kovin tuttu, kun en käsittele/tilaa noita paketteja. Kysäisin meillä sisäisesti asiasta, ei olleet muutkaan juuri tuohon termiin törmänneet. Meillä pohdittiin voisiko termi liittyä jotenkin kokoomateoksiin: https://www.kiwi.fi/display/rda/Aggregaatit+ja+diakroniset+teokset?preview=/274235824/303300938/Aggregaatit.pdf

**Reply by hanhisalok** [reply]
Kuvailussa näytetään käytettävän sanaa aggregaatti ihan suomeksikin...tai sitten joku tuolla mainittu "teosten kooste". Mutta miten se "full" siihen vielä osuisi. Tarkoittaisikohan se että kokoteksti verkossa saatavana.

**Reply by AnneliO** [reply]
Koha-manuaalissa kerrotaan näin:

ERM_PACKAGE_CONTENT_TYPE

Used to describe the types of material found in a local package in the [e-resources management module](https://koha-community.org/manual/23.05/en/html/erm.html#erm-label).

For local packages only

Default values are

Aggregated full (AggregatedFullText)
Abstract and index (AbstractAndIndex)
E-book (EBook)
E-journal (EJournal)
Mixed content (MixedContent)
Online reference (OnlineReference)
Print (Print)
Streaming media (StreamingMedia)
Unknown (Unknown)

Eli vissin se jotain kokoteksteihin liittyvää on. :D

**Reply by inkerih** [reply]
Melinda-sanastossa puhutaan aggregaattien olevan yhdistelmiä tai koosteita. https://www.kiwi.fi/display/melinda/Melinda-sanasto

Voisin ehkä laittaa tämän ja jokusen muun aihetta liippaavan linkin tuonne termistö-sivulle?
